service/mantle/shopify/ApiServices.xml

```diff
@@ -1038,7 +1038,7 @@
                         </entity-find>
 
                         <if condition="productIdList.size() == 0">
-                            <return error="true" message="ERROR: SHOPIFY VARIANT ID ${orderItem.variant_id} IS NOT REGISTERED IN PRODUCT IDENTIFICATION, CONTACT THE ADMIN"/>
+                            <return error="true" message="ERROR: THE PRODUCT ID ${orderItem.variant_id} ON SHOPIFY ORDER ${order.name} IS NOT REGISTERED ON DB, CONTACT THE ADMIN FOR CREATING ONE IN PRODUCTS SCREEEN"/>

```

Comment:
Change message to: THE PRODUCT ID ${orderItem.variant_id} ON SHOPIFY ORDER ${order.name} IS NOT REGISTERED ON DB, PLEASE CONTACT ADMIN.

Reply:
done!